src/main/kotlin/LoggerUtils.kt

```diff
@@ -0,0 +1,207 @@
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.classic.spi.ThrowableProxyUtil
+import ch.qos.logback.core.CoreConstants
+import ch.qos.logback.core.LayoutBase
+import org.apache.commons.text.StringEscapeUtils
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.net.InetAddress
+import java.text.SimpleDateFormat
+import java.util.*
+
+private const val UNSET_TEXT = "NOT_SET"
+private val defaultFormat = makeUtcDateFormat() // 2001-07-04T12:08:56.235
+
+private var hostname = InetAddress.getLocalHost().hostName
+private var environment = System.getProperty("environment", UNSET_TEXT)
```

Comment:
We don't set any system properties in K2HB I think, we send in as env vars.

Reply:
Used env vars. Incoming change in aws-ingest to set these env vars.